aten/src/ATen/core/dispatch/Dispatcher.h

```diff
@@ -144,6 +144,8 @@ class CAFFE2_API Dispatcher final {
   std::list<OperatorDef> operators_;
   LeftRight<ska::flat_hash_map<OperatorName, OperatorHandle>> operatorLookupTable_;
   impl::KernelFunctionTable backendFallbackKernels_;
+  // Set of backends which have specified they want fallthrough behavior
```

Comment:
I think it's the set of backends that **don't** have specified they want fallthrough behavior.